installer/templates/phx_single/config/dev.exs

```diff
@@ -7,7 +7,9 @@ import Config
 # watchers to your application. For example, we use it
 # with webpack to recompile .js and .css sources.
 config :<%= app_name %>, <%= endpoint_module %>,
-  http: [port: 4000],
+  # Binding to loopback ipv4 address prevents access from other machines.
+  # Change to `ip: {0, 0, 0, 0}` to allow access from other machines.
```

Comment:
After "to allow access from other machines", I would add "or from the host machine if Phoenix is running in a VM or container""

Reply:
Running in vm / container does not necessarily requires binding on public interface - e.g. users may run proxy on the machine - in that case it runs in container/vm but does require that... but I'd like to add that as an "example" (not another case with "or")  :heart:.

Could you help improve the wording?

> Change to `ip: {0, 0, 0, 0}` to allow access from other machines,
> such as accessing the app runnining in a virtual machine or container
> from the host machine if Phoenix is running in a VM or container

Also.. I'm not sure "Phoenix is running" is right statement. Do we have similar comments somewhere? We're running this app, not "Phoenix".